compose/common.yml

```diff
@@ -69,6 +76,24 @@ x-provider-server-environment: &provider-server-environment
   USE_BLOCKFROST: ${USE_BLOCKFROST:-false}
   USE_SUBMIT_API: ${USE_SUBMIT_API:-false}
 
+x-provider-server-environment-blockfrost: &provider-server-environment-blockfrost
```

Comment:
The `x-*` sections are needed to reduce code repetition to group settings required by more containers.
Better to define these configurations (removing duplicates) directly in `provider-server-blockfrost` container configuration.

Reply:
Docker-related changes are no longer a part of this PR.